.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "pip" # See documentation for possible values
+    directory: "/docker/jupyterlab" # Location of package manifests
+    schedule:
+      interval: "weekly"
+    groups:
+      github-actions-dependencies:
```

Comment:
This line represents the name of the grouped dependencies and is used in the pull request name, so should probably be something like `jupyterlab-included-packages`